scripts/test/e2e/run

```diff
@@ -69,7 +71,7 @@ runtests() {
         GOPATH="$GOPATH" \
         PATH="$PWD/build/:/usr/bin:/usr/local/bin:/usr/local/go/bin" \
         HOME="$HOME" \
-        DOCKER_CLI_E2E_PLUGINS_EXTRA_DIRS="$PWD/build/plugins-linux-amd64" \
+        DOCKER_CLI_E2E_PLUGINS_EXTRA_DIRS="$PWD/build/plugins-linux-${GOARCH}" \
```

Comment:
Had to double-check but looks like we run this script with `set -eu -o pipefail`, so it should fail if `GOARCH` is not set for whatever reason 👍

Reply:
Yup! This was a pain to deal with, and that seemed the easiest way to get `GOARCH` in there so I could run it locally as well.